sass/chosen.scss

```diff
@@ -1,6 +1,35 @@
 $chosen-sprite: url('chosen-sprite.png') !default;
 $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
 
+$chosen-bg-color: #fff;
+$chosen-box-shadow-color: #000;
+$chosen-border-color: #aaa;
+$chosen-group-name-color: #999;
+$chosen-highlighted-color: #3875d7;
+$chosen-highlighted-second-color: #2a62bc;
+
+$chosen-search-choice-bg-color: #eee;
+$chosen-search-choice-disabled-bg-color: #e4e4e4;
+$chosen-search-choice-color: #333;
+$chosen-search-choice-bg: linear-gradient(#f4f4f4 20%, #f0f0f0 50%, #e8e8e8 52%, #eee 100%);
+$chosen-search-choice-disabled-color: #666;
+$chosen-search-choice-focus-bg-color: $chosen-search-choice-disabled-color;
+$chosen-search-choice-disabled-border-color: #ccc;
+$chosen-no-results-bg: #f4f4f4;
+
+$chosen-color: #222;
+$chosen-border-color: #5897fb;
+$chosen-single-color: #444;
+$chosen-bg: linear-gradient($chosen-bg-color 20%, #f6f6f6 50%, #eee 52%, #f4f4f4 100%);
+$chosen-single-bg: linear-gradient(#eee 20%, $chosen-bg-color 80%);
+$chosen-default-color: $chosen-group-name-color;
+$chosen-container-color: $chosen-single-color;
+$chosen-container-disabled-color: $chosen-search-choice-disabled-border-color;
+$chosen-container-no-results-color: #777;
+$chosen-choices-color: #999;
+$chosen-choices-bg: linear-gradient($chosen-search-choice-bg-color 1%, $chosen-bg-color 15%);
+$chosen-drop-result-selected: $chosen-search-choice-disabled-border-color;
```

Comment:
I'm not sure how I feel about using a variable inside the definition of another variable. Normally I'd be all for it, but in this case (and a few other cases I see in this file), the variables are completely unrelated. For instance, `$chosen-drop-result-selected` has absolutely nothing to do with `$chosen-search-choice-disabled-border-color` (other than being a similar hex color), and by making the former defined by the latter, the code is making the two seem related in some way. I'll let another dev make a call on that (cc @stof)